far/string_utils.cpp

```diff
@@ -279,61 +279,58 @@ std::optional<std::pair<size_t, size_t>> icase_searcher::find_in(string_view con
 	return m_Searcher.find_in(m_HayStack, Reverse);
 }
 
-static void normalize_for_search(string_view const Str, string& Result, string& Intermediate, std::vector<WORD>& Types)
+namespace
```

Comment:
I don't mind replacing static with an anonymous namespace, but it requires extra indentation and makes the diff larger and a bit harder to review.

If you're touching it anyway, maybe merge the whole thing into `fuzzy_searcher_impl::normalize`? It is only used there and in the test below.

Reply:
> If you're touching it anyway, maybe merge the whole thing into `fuzzy_searcher_impl::normalize`?

I had this at some point, then stumbled upon the test and thought that maybe the policy call for testing of such functions in strict isolation. I will happily apply this change.

Reply:
> maybe the policy call for testing of such functions in strict isolation

Not really, more like "the test should work without initializing the rest of the codebase, e.g. config, language etc."
Sometimes it's easier to do so by extracting some code to free functions, but in this case the class is trivial and you've already added fuzzy_searcher_impl to the test body, so why not.

Reply:
Done!